api/proto/teleport/legacy/types/types.proto

```diff
@@ -1661,6 +1663,57 @@ message ProvisionTokenSpecV2TerraformCloud {
   string Hostname = 3 [(gogoproto.jsontag) = "hostname,omitempty"];
 }
 
+message ProvisionTokenSpecV2Bitbucket {
+  // Rule is a set of properties the Bitbucket-issued token might have to be
+  // allowed to use this ProvisionToken.
+  message Rule {
+    // WorkspaceUUID is the UUID of the workspace for which this token was
+    // issued. Bitbucket UUIDs must begin and end with braces, e.g. `{...}`.
+    // This value may be found in the Pipelines -> OpenID Connect section of the
+    // repository settings.
+    string WorkspaceUUID = 1 [(gogoproto.jsontag) = "workspace_uuid,omitempty"];
+
+    // RepositoryUUID is the UUID of the repository for which this token was
+    // issued. Bitbucket UUIDs must begin and end with braces, e.g. `{...}`.
+    // This value may be found in the Pipelines -> OpenID Connect section of the
+    // repository settings.
+    string RepositoryUUID = 2 [(gogoproto.jsontag) = "repository_uuid,omitempty"];
+
+    // PipelineUUID is the UUID of the pipeline for which this token was issued.
+    // Bitbucket UUIDs must begin and end with braces, e.g. `{...}`
+    string PipelineUUID = 3 [(gogoproto.jsontag) = "pipeline_uuid,omitempty"];
+
+    // StepUUID is the UUID of the pipeline step for which this token was
+    // issued. Bitbucket UUIDs must begin and end with braces, e.g. `{...}`
+    string StepUUID = 4 [(gogoproto.jsontag) = "step_uuid,omitempty"];
```

Comment:
I think I'm going to drop these 2 fields from the proto spec. They appear to be unique for every CI run so they're only useful as logged OIDC claims to identify the particular run/step.